setup-chainguard-terraform/action.yaml

```diff
@@ -31,14 +32,16 @@ runs:
   steps:
     - uses: chainguard-dev/actions/setup-chainctl@main
       with:
+        environment: ${{ inputs.environment }}
+        audience: ${{ inputs.audience }}
         invite-code: ${{ inputs.invite-code }}
 
     - shell: bash
       run: |
         cat > ~/.terraformrc <<EOF
         provider_installation {
           network_mirror {
-            url = "https://storage.googleapis.com/us.artifacts.${{ inputs.environment }}.appspot.com/terraform-provider/"
+            url = "https://dl.${{ inputs.environment }}/terraform-provider/"
```

Comment:
In the downstream action, we are passing the gcp project, if we stick with this we will need to make some changes in the mono to adjust this
 before merge this let  me check and make a pr to update the mono side